content/courses/offline-transactions/durable-nonces.md

```diff
@@ -14,39 +14,39 @@ description: "Use durable nonces to sign transactions ahead of time."
   have an expiration date of 150 blocks (~80-90 seconds).
```

Comment:
@darkvallen  you also have https://github.com/solana-foundation/developer-content/pull/450 which is for durable-nonces.md. It makes it **very** difficult for us to track your work when this happens. 

Please put all your changes to durable-nonces.md in that PR and remove it from this PR.

Reply:
Fixed.
https://github.com/solana-foundation/developer-content/pull/450 Ready for further review.